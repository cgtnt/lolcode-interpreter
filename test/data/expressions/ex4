ALL OF WIN AN WIN AN WIN MKAY
ALL OF WIN AN FAIL MKAY
ANY OF FAIL AN FAIL AN WIN MKAY
ANY OF FAIL AN FAIL MKAY
