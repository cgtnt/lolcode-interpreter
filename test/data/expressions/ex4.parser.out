ALL OF ALL OF True True True